modules/context/org.go

```diff
@@ -129,7 +129,11 @@ func HandleOrgAssignment(ctx *Context, args ...bool) {
 
 	// Team.
 	if ctx.Org.IsMember {
-		if ctx.Org.IsOwner {
+		maxPerm, err := org.GetOrgUserMaxAuthorizeLevel(ctx.User.ID)
+		if err != nil {
+			ctx.ServerError("GetOrgUserMaxAuthorizeLevel", err)
+		}
+		if ctx.Org.IsOwner || maxPerm >= perm.AccessModeAdmin {
```

Comment:
I think we should check if the user is in a team with admin permission and include all repositories in this organization.